### DISCUSSION TITLE
PWA not showing new look

### DISCUSSION THREAD

**Discussion by yssunjoko** [quasarframework/quasar#16271] [CLI - PWA mode]
Hi, I'm building a Quasar PWA apps whose build result I then put into Laravel blade through Laravel mix.
Everything works fine, except that whenever I push another new look and mix it into Laravel Blade, the old look remains. It doesn't turn into the new one, unless I press Ctrl+F5.

I am a newbie in PWA, I love PWA because it's installable directly into mobile phone / desktop, but I have no idea about how to handle this change issue. What's the problem?
Please give me some hint.

**Comment by tuxsisir** [comment]
@yssunjoko Did you find anything on this?

**Reply by stefanvanherwijnen** [reply]
You need to explicitly tell a PWA to reload, for example by CTRL+F5.
https://quasar.dev/quasar-cli-webpack/developing-pwa/configuring-pwa#reload-and-update-automatically

It's advised to display a notification on the updated() event and reload the page after clicking a button:
https://quasar.dev/quasar-cli-webpack/developing-pwa/handling-service-worker#interacting-with-service-worker

https://github.com/quasarframework/quasar/blob/a0ddd490d6851160f73d1bd012b43e98e15b7508/docs/src-pwa/register-service-worker.js
